src/coinbase/address.ts

```diff
@@ -79,6 +80,41 @@ export class Address {
     return Balance.fromModelAndAssetId(response.data, assetId).amount;
   }
 
+  /**
+   * Returns the historical balance of the provided asset.
+   *
+   * @param assetId - The asset ID.
+   * @returns The list of historical balance of the asset.
+   */
+  public async listHistoricalBalance(assetId: string): Promise<HistoricalBalance[]> {
+    const historyList: HistoricalBalance[] = [];
+    const queue: string[] = [""];
+
+    while (queue.length > 0) {
```

Comment:
does this method list historical balances from genesis?

Should we add paging support so we are not waiting on so many balances before returning the array?

Reply:
added, thanks for catching!